docs/projects/pulsar_adc/index.rst

```diff
@@ -152,21 +152,40 @@ PulSAR_ADC_FMC
 Configuration modes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For AD40xx/ADAQ40xx projects the AD40XX_ADAQ400X_N parameter defines the type
-of ADC (AD40xx or ADAQ40xx). By default is set to 1. Depending on the project,
-some hardware modifications need to be done on the board and/or make command:
+The Zedboard based Pulsar ADC project can be built to use either the FMC 
+connector or the Pmod JA connector. The FMC_N_PMOD parameter is used to select
+between the two options. By default it is set to 1 (FMC).
 
-In case of the **ADAQ40xx** project:
+The Pulsar project also supports more exotic configurations required for
+certain ADCs like the AD7944. These modes are selected using the SPI_OP_MODE 
+parameter.
+
+Build examples:
+
+Zedboard Pmod support
+
+.. code-block::
+
+   make FMC_N_PMOD=0
+
+Zedboard standard configuration FMC support (default)
+
+.. code-block::
+
+   make FMC_N_PMOD=1 SPI_OP_MODE=0 - builds standard FMC version
+
+Zedboard FMC support for AD7944 4-wire mode ("multi")
 
 .. code-block::
 
-   make AD40XX_ADAQ400X_N=0
+   make FMC_N_PMOD=1 SPI_OP_MODE=1
 
-In case of the **AD40xx** project:
+Zedboard FMC support for AD7944 chain mode or 3-wire "single"
 
 .. code-block::
 
-   make AD40XX_ADAQ400X_N=1
+   make FMC_N_PMOD=1 SPI_OP_MODE=2
+
```

Comment:
Build examples should be moved under the `Building the project section`, and in the `Configuration modes` just to list the possible values

Reply:
Done